R/GitHostGitHub.R

```diff
@@ -88,6 +88,25 @@ GitHostGitHub <- R6::R6Class(
       private$endpoints$repositories <- glue::glue("{private$api_url}/repos")
     },
 
+    # Set owner type
+    set_owner_type = function(owners) {
```

Comment:
When token doesn't have required scope, code crashed with unformative error instead of error: 

`✔ Set connection to GitHub.
ℹ [Host:GitHub][Engine:GraphQl][Scope:ddsjoberg] Pulling repositories...
Called from: graphql_engine$get_repos_from_org(org = org, type = type)
Browse[1]> c
Error in `purrr::map()`:
ℹ In index: 1.
Caused by error in `purrr::map()`:
ℹ In index: 1.
Caused by error in `UseMethod()`:
! no applicable method for 'filter' applied to an object of class "NULL"
Run `rlang::last_trace()` to see where the error occurred.`

![image](https://github.com/user-attachments/assets/4892d0a7-e1b9-4217-a92e-e546d4ed337d)

Reply:
I will create a separate issue on that, as it is more generic problem, which should be solved when setting up hosts.